src/ifp/include/ifp/InitFloorplan.hh

```diff
@@ -35,8 +35,9 @@
 
 #pragma once
 
+#include <set>
 #include <string>
-
+#include <vector>
```

Comment:
I don't see a usage of vector in this header.  It is needed elsewhere please add it to the file(s) that use it.